client/state/stats/lists/test/utils.js

```diff
@@ -572,6 +572,7 @@ describe( 'utils', () => {
 					)
 				).toEqual( [
 					{
+						id: 0,
```

Comment:
Just a heads up that the home page post ID isn't always going to be 0 (you can switch any post with a post_type of `page` to be the home page too, and these will all have an ID)

Also there are other URLs, like taxonomy pages or the blog page, that also might not have IDs.